routing/router.go

```diff
@@ -1069,9 +1069,8 @@ func (r *ChannelRouter) processUpdate(msg interface{}) error {
 		r.channelEdgeMtx.Lock(msg.ChannelID)
 		defer r.channelEdgeMtx.Unlock(msg.ChannelID)
 
-		edge1Timestamp, edge2Timestamp, exists, err := r.cfg.Graph.HasChannelEdge(
-			msg.ChannelID,
-		)
+		edge1Timestamp, edge2Timestamp, exists, err :=
```

Comment:
Prefer the prior line style wise even if it was over the col limit IMO.

Reply:
Not sure I agree but OK 😛